roachtest: add ruby-pg failing tests to blocklist; fix gorm

rafiss opened the pull request:
fixes https://github.com/cockroachdb/cockroach/issues/67701
fixes https://github.com/cockroachdb/cockroach/issues/67455

Release note: None

rafiss committed:
roachtest: add ruby-pg failing tests to blocklist

Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/67789)
<!-- Reviewable:end -->

rafiss committed:
roachtest: fix gorm roachtest and update

Release note: None

 Please enter the commit message for your changes. Lines starting

rafiss commented:
tftr!

bors r=RichardJCai
